src/components/RehydrationProvider.js

```diff
@@ -34,10 +33,7 @@ const transformRemoveFetchErr = createTransform(state => omit(state, ['isFetchin
 })
 
 export default class RehydrationDelayedProvider extends Component {
-  constructor(props) {
-    super(props)
-    this.state = { rehydrated: false }
-  }
+  state = { rehydrated: false }
```

Comment:
Actually i don't think it was required to do refactor of code like this

Reply:
I must have missed this. I'll revert this once I manage to fix the conflicts